aladdin/commands/build.py

```diff
@@ -18,5 +19,9 @@ def build_args(args):
 
 @container_command
 def build(build_args):
+    tag = "local"
+    k3d = K3d()
     pc = ProjectConf()
-    pc.build_docker(env={"HASH": "local"}, build_args=build_args)
+    pc.build_docker(env={"HASH": tag}, build_args=build_args)
+    images = pc.get_docker_images()
+    k3d.import_images([f"{image}:{tag}" for image in images])
```

Comment:
locally built images aren't available to k3d without importing them. This causes `aladdin build` to take a little bit longer (~15secs/image)

Reply:
15 seconds, ouch

Reply:
the plus side is though that we should be building way less, since the images will stay on the host docker instance